f/briefing/f_orbatNotes.sqf

```diff
@@ -153,3 +153,17 @@ if (count _veharray > 0) then {
 // Insert final result into subsection ORBAT of section Notes
 waitUntil {scriptDone f_script_briefing};
 player createDiaryRecord ["diary", ["ORBAT", _orbatText]];
+
+// Add button to take control of group when Arma does not update leadership correctly.
+player createDiaryRecord ["fa3_actions", ["Group Leadership",
```

Comment:
As a tiny point, I'd maybe put this elsewhere e.g. in it's own briefing file (or own component entirely). But so small it's fine here